test/Vagrantfile

```diff
@@ -94,8 +94,8 @@ Vagrant.configure("2") do |config|
             inline: "ip -6 a a fd05::1/96 dev enp0s9 || true"
 
         # This network is only used by NFS
-        server.vm.network "private_network", type: "dhcp"
-        server.vm.synced_folder cilium_dir, cilium_path, nfs: $NFS
+        server.vm.network "private_network", ip: "192.168.38.10", bridge: "enp0s10"
```

Comment:
Should this be guarded by an `if ENV["NFS"]` as we are doing for the dev VMs?

Reply:
Yep, I'm planning to do one more iteration on this PR to fix that and other nits before opening for everyone's reviews.